02/PrintRecords.cpp

```diff
@@ -0,0 +1,133 @@
+#include <cassert>
+#include <fstream>
+#include <iomanip>
+#include <iostream>
+#include <string>
+#include "PrintRecords.h"
+
+using namespace std;
+
+const int STRING_SIZE = 10;
+record Info;
```

Comment:
no global variables here... structure this example this way

ManageRecordsExample()
1. this will prompt user 
 a) enter new record
 b) update existing record
(every time user enters a new record or updates it displays full records on the screen)